npm/cypress-schematic/src/schematics/ng-add/files/cypress/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "<%= relativeToWorkspace %>/tsconfig.json",
-  "include": ["**/*.ts"],
+  "include": ["**/*.cy.ts"],
```

Comment:
We should include `spec` as well

Reply:
[This entry](https://docs.cypress.io/guides/tooling/typescript-support#Configure-tsconfig-json) in the docs seems to suggest that we should include all `.ts.` files with `"**/*.ts"`. Has that changed?